example/README.md

```diff
@@ -0,0 +1,95 @@
+# Conquer Club 
+### Event sourced DDD example
+
+On the [Conquer Club](https://www.conquerclub.com/) website you can play a game
+that is inspired by the board game RISK�. In this demo, (most of) the rules of
+that game have been implemented using Qowaiv's Aggregate Root, to give an
+impression on how a (not trivial) event sourced DDD solution could look like.
+
+## Always valid
+The validness of the game (state) is guarded by a validator that is injected
+via the constructor:
+``` C#
+ public Game(GameId id) : base(id, new GameValidator()) { }
+```
+
+The `[GameValidator](ConquerClub.Domain/Validation/GameValidator.cs)` uses
+the [Qowaiv.Validation.Fluent](https://github.com/Qowaiv/qowaiv-validation), an
```

Comment:
add 'package/library' after the package name or remove 'the' before the package name